flow-preferences-tests/src/androidTest/java/com/fredporciuncula/flow/preferences/tests/PreferenceTest.kt

```diff
@@ -1,11 +1,12 @@
 package com.fredporciuncula.flow.preferences.tests
 
 import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.CompletableDeferred
 import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.flow.emitAll
-import kotlinx.coroutines.flow.first
-import kotlinx.coroutines.flow.flow
-import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.flow.*
```

Comment:
🔍 Would you mind moving away from the wildcard import? The current static analysis in place will actually break the build with it.